iml-gui/crate/src/components/breadcrumbs.rs

```diff
@@ -0,0 +1,103 @@
+use crate::{Msg, Page};
+
+use seed::{
+    a, attrs, div,
+    dom_types::{At, El, Node, UpdateEl, View},
+    li, ol,
+    prelude::IndexMap,
+};
+
+use std::cmp::PartialEq;
+use std::collections::LinkedList;
+
+pub struct Model<Crumb> {
+    crumbs: LinkedList<Crumb>,
+}
+
+impl<Crumb> Default for Model<Crumb> {
```

Comment:
Could this not be derived?

Reply:
I think it can. Probably it's left from previous ideas.